onnxruntime/core/providers/shared_library/provider_interfaces.h

```diff
@@ -142,6 +142,10 @@ struct Node__EdgeIterator {
 struct ProviderHost {
   virtual const OrtApiBase* OrtGetApiBase() = 0;
 
+  virtual Status GetEPOptimizerByName(const std::string& name,
```

Comment:
nit: Sounds like this optimizes an EP. Could we call it `GetOptimizerByName`?

Reply:
changed to GetOptimizerByName